src/typescript/Scala.tmLanguage.ts

```diff
@@ -425,7 +425,7 @@ export const scalaTmLanguage: TmLanguage = {
     'using': {
       patterns: [
         {
-          match: `(?<=\\()\\s*(using)\\s+(?=[\\w\\d\\(\\.\\(\\{'"])`,
+          match: `(?<=\\()\\s*(using)\\s+(?=[\\w\\d\\(\\.\\(\\{'"$])`,
```

Comment:
Is there a reason for why we have the positive lookahead `(?=[\\w\\d\\(\\.\\(\\{'"])`? 

I'm also noticing here that we have `\\(` twice in the character set `[\\w\\d\\(\\.\\(\\{'"$]`. 

Also, could some of these be un-escaped? I think `(` and `{` don't need to be escaped in character sets.

Reply:
The positive lookahead seems to be there for historical reasons. I simplified the regex to not use it at all and added some tests for the cases the lookahead was trying to avoid.